### ISSUE TITLE
Als leverancier wil ik weten wat de verhouding is tussen zaak(type) en verzoek

### ISSUE THREAD

**Issue by hdksi** [VNG-Realisatie/gemma-zaken#1694]
...ik mijn afhandelapplicatie zondig kan inrichten dat wordt voldaan aan (de architectuur van) de API-standaarden voor Zaakgericht werken.

...met aandacht voor het feit dat bijvoorbeeld twee afhandelapplicaties verschillende processen kunnen definiëren voor één zaaktype, waardoor mutatie door de ene applicatie kan leiden tot problemen in de andere.

**Comment by joeribekker** [comment]
Hugo gaf nog aan dat: "Een proces moet een zaak aansturen volgens het zaaktype"

**Comment by joeribekker** [comment]
> en aanvullende procesgegevens in de afhandelapplicatie kunnen zijn gedefinieerd. Dit levert problemen op als bijvoorbeeld twee afhandelapplicaties verschillende processen hebben gedefinieerd voor één zaaktype

Dit kan eigenlijk gewoon niet. Je hebt dan immers eigenlijk een proces die OVER 2 afhandelapplicaties gaat. Dan kan niet slechts 1 van de afhandelapplicaties HET proces hebben.

**Comment by hdksi** [comment]
@joeribekker klopt. Bij nader inzien is dit geen goede illustratie van de casus.

**Comment by EdwinCoster** [comment]
Is het probleem niet dat in een afhandelcomponent het proces als zaak wordt gedefinieerd? En ingericht?

**Comment by sergei-maertens** [comment]
In de praktijk zien we dat taakapplicaties aanvullende procesgegevens bijhouden, met een referentie naar de zaak waar ze bij horen.

Bij Utrecht gebruiken we Camunda processen. De procesvariabelen leven daar dus in de database van de proces-engine, en deze kunnen uitgelezen worden door de afhandelcomponenten. In het proces zit dan bijvoorbeeld een BSN, waarmee we in het afhandelcomponent de bijhorende BRP record zoeken.

Daarnaast hebben we bijvoorbeeld ook proces-specifieke accorderingen, adviezen en ad-hoc activiteiten die in eigen registraties bestaan, maar die zijn allemaal op een zaak gerelateerd.
